hw/xbox/dsp/dsp_cpu.c

```diff
@@ -26,6 +26,8 @@
 #include <string.h>
 #include <assert.h>
 
+#include "qemu/osdep.h"
+#include "qemu/bswap.h"
```

Comment:
We probably want our own endianess swap in this file, because it belongs to Hatari, not QEMU.

Reply:
I think in general we should prefer to avoid having duplicate platform abstraction functions, this code lives in XQEMU after all. That said, it would be nice if upstream could benefit improvements we make (we want to be good open-source citizens)--in this case I would be okay with something like that.